lib/exportCsv.js

```diff
@@ -62,6 +62,7 @@ class FieldList {
 }
 
 export default function exportToCsv(objectArray, opts) {
+  console.warn('[DEPRECATION] - use "@folio/stripes/components/exportToCsv" instead.');
```

Comment:
Surely it's inconsistent with standard Node-package style to emit a single-line message? The usual procedure is a spew several hundred lines of React-internals stack-trace all over the JS console. Also, it seems this message only appears if someone actually triggers the function. Can you somehow arrange for it to appear on every render of a component that includes a mention of this function? Otherwise there is a very real danger that the JS console will be clear enough to be useful.